guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,21 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Don't use string interpolation
+
+By using string interpolation it is impossible for logging frameworks that support structure logging to distinguish values within the string. We must use the normal argument based overload so logging frameworks like Serilog can capture structured data.
+
+Wrong:
+
+```c#
+logger.Info("File share data bus started. Location: " + basePath);"
```

Comment:
string concatenation or interpolation, both result in the fact that the logger will not get the arguments if it wanted to apply structured logging.